main.go

```diff
@@ -13,6 +13,7 @@ const SOURCE string = `source`
 const TARGET string = `target`
 const RESOLUTION string = `resolution`
 const PAGESIZE string = `pagesize`
+const KEEP_EXTERIORRING string = `keepexterior`
```

Comment:
```suggestion
const KEEP_OUTER_RINGS string = `keepexterior`
```
consequent exterior of outer, meervoud, spatie in het engels

Reply:
tsja, exterior ring wordt breder gebruikt outerring vind ik een betere naam eigenlijk

Reply:
aangepast

Reply:
Ik snap deze aanpassing niet, waarom deze wijziging?